Async addition (and cleanup)

Zwork101 opened the pull request:
I kinda changed like, a lot, which wasn't my first intention, but that's
how it turned out. I moved some files, added async, cleaned a lot of
stuff up. However, it should all be still backwards compatible.

phoemur commented:
Do you feel like assuming this repo ?

I'm in other projects actually and won't maintain it anymore...

Tx

Em sex, 5 de out de 2018 às 22:32, Nate the great <notifications@github.com>
escreveu:

> I kinda changed like, a lot, which wasn't my first intention, but that's
> how it turned out. I moved some files, added async, cleaned a lot of
> stuff up. However, it should all be still backwards compatible.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/phoemur/ipgetter/pull/32
> Commit Summary
>
>    - Async addition
>
> File Changes
>
>    - *M* MANIFEST
>    <https://github.com/phoemur/ipgetter/pull/32/files#diff-0> (1)
>    - *D* ipgetter.py
>    <https://github.com/phoemur/ipgetter/pull/32/files#diff-1> (176)
>    - *A* ipgetter/__init__.py
>    <https://github.com/phoemur/ipgetter/pull/32/files#diff-2> (181)
>    - *A* ipgetter/async.py
>    <https://github.com/phoemur/ipgetter/pull/32/files#diff-3> (92)
>    - *M* setup.py
>    <https://github.com/phoemur/ipgetter/pull/32/files#diff-4> (6)
>
> Patch Links:
>
>    - https://github.com/phoemur/ipgetter/pull/32.patch
>    - https://github.com/phoemur/ipgetter/pull/32.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/phoemur/ipgetter/pull/32>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGaTrgQVdN1q1cpG2O6DZflEYl5bz2VCks5uiAgqgaJpZM4XLHoe>
> .
>

Zwork101 commented:
To be completely honest with you, I'd rather make it from the ground up. I
disagree with a lot of methods you use in this repo, so no thanks :/

On Sat, Oct 6, 2018, 1:04 PM phoemur <notifications@github.com> wrote:

> Do you feel like assuming this repo ?
>
> I'm in other projects actually and won't maintain it anymore...
>
> Tx
>
> Em sex, 5 de out de 2018 às 22:32, Nate the great <
> notifications@github.com>
> escreveu:
>
> > I kinda changed like, a lot, which wasn't my first intention, but that's
> > how it turned out. I moved some files, added async, cleaned a lot of
> > stuff up. However, it should all be still backwards compatible.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/phoemur/ipgetter/pull/32
> > Commit Summary
> >
> > - Async addition
> >
> > File Changes
> >
> > - *M* MANIFEST
> > <https://github.com/phoemur/ipgetter/pull/32/files#diff-0> (1)
> > - *D* ipgetter.py
> > <https://github.com/phoemur/ipgetter/pull/32/files#diff-1> (176)
> > - *A* ipgetter/__init__.py
> > <https://github.com/phoemur/ipgetter/pull/32/files#diff-2> (181)
> > - *A* ipgetter/async.py
> > <https://github.com/phoemur/ipgetter/pull/32/files#diff-3> (92)
> > - *M* setup.py
> > <https://github.com/phoemur/ipgetter/pull/32/files#diff-4> (6)
> >
> > Patch Links:
> >
> > - https://github.com/phoemur/ipgetter/pull/32.patch
> > - https://github.com/phoemur/ipgetter/pull/32.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/phoemur/ipgetter/pull/32>, or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AGaTrgQVdN1q1cpG2O6DZflEYl5bz2VCks5uiAgqgaJpZM4XLHoe
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/phoemur/ipgetter/pull/32#issuecomment-427590063>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AYdtrtq6eOiAn8qZspNyz_WvJH-KgulBks5uiOKqgaJpZM4XLHoe>
> .
>
